keyboards/jlw/vault35_katana/keymaps/vial/rules.mk

```
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+VIALRGB_ENABLE = yes
```

```suggestion
VIALRGB_ENABLE = yes
ENCODER_MAP_ENABLE = yes
```
Ideally, encoder mapping goes in each keymap's `rules.mk`, not in the `config.h` for the entire keyboard.